transposon/analyzeBCcounts.sh

```diff
@@ -172,7 +172,7 @@ if [ "${minCellBCLength}" -gt 0 ]; then
         #June 2020 scaled
         scRNAseqbase="/vol/mauranolab/transposon/scrnaseq/merged_T0219toT0222"
         #/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_conflicting_cells.txt
-        genotypeClonesArgs="--blacklist /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_multisite.txt,/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_shared.txt --annotateclones /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/bc_transfection-occurrence.txt.gz"
+        genotypeClonesArgs="--blacklist /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_multisite.txt,/vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/blacklist_shared.txt --annotateclones /vol/mauranolab/ribeia01/transposon_10x/T0219toT0221/bc_transfection-occurrence.txt.gz --transfectionKey transfection --removeMinorityBCsFromConflictingCells 0.20 --removeConflictingCells"
```

Comment:
Should I include `--minPropOfBCReads 0.10`? Did we decide to change `--minPropOfCellReads`?

Reply:
Let's leave minPropOfBCReads/minPropOfCellReads as-is; I will run once to see how things behave

We also may not want to change them for T0190, which is where I first tuned them